tests/integration/multi_table/test_hma.py

```diff
@@ -1098,3 +1099,23 @@ def test_hma_numerical_distributions(self):
         assert data.keys() == samples.keys()
         for table_name, table in samples.items():
             assert set(data[table_name].columns) == set(table.columns)
+
+    def test_get_learned_distributions_error_msg(self):
```

Comment:
should this be a unit test?

Reply:
No because I am calling `fit` directly, to be a proper unit test I'd have to mock it (I can do that if you think it's warranted).